torch/_dynamo/variables/torch.py

```diff
@@ -82,6 +82,8 @@
     torch.is_floating_point,
     torch.nn.functional._Reduction.get_enum,
     torch._C._get_privateuse1_backend_name,
+    torch.distributed.get_rank,
+    torch.distributed.get_world_size,
```

Comment:
basic Q: it seems a bit redundant to force dynamo to inline a hardcoded list of torch API's into the graph, if we know that AOTAutograd will do it later anyway. Is main purpose to (mildly) limit the API surface for any backends that want to do stuff before AOTAutograd?

Reply:
It's actually not inlining; this says that it's OK to run the operation, get out a Python constant, and treat it as constant for the purposes of the rest of Dynamo execution

Reply:
ah got it. thanks!